### PR TITLE
ci: upload release assets

### PR DESCRIPTION
**Pull request by naveen521kk**
## List of Changes
This was a request from AUR package maintainer to upload the release asset to Github Release.

## Explanation for Changes
Using action/upload-release-asset action this is easy.

## Acknowledgement
- [x] I have read the [Contributing Guidelines](https://github.com/ManimCommunity/manim/wiki/Documentation-guidelines-(WIP))

### CODE REVIEW

File: .github/workflows/python-publish.yml [hunk 1]
```diff
@@ -26,3 +26,26 @@ jobs:
...
       run: poetry publish --build
+
+    - name: Get Upload URL
+      id: create_release
+      shell: bash
+      env:
+        access_token: ${{ secrets.GITHUB_TOKEN }}
+        tag_act: ${{ github.ref }}
+      run: |
+          ref_tag=$(python -c "print('${tag_act}'.split('/')[-1])")
```
Comment (eulertour) [inline .github/workflows/python-publish.yml:37]: What is ${tag_act}?

**Comment by eulertour** [comment]
This is fine to add, but given how niche a use it is and the complexity of the action you'll likely have to maintain it yourself.

File: .github/workflows/python-publish.yml (hunk 1 above)
Comment (naveen521kk) [reply .github/workflows/python-publish.yml:37]: That would have some value on which tag it runs for example `refs/origin/v0.1.0`.

**Comment by naveen521kk** [comment]
I don't find any better way to get upload url. Or another thing which can be done is you just create a tag and this will release instead of you creating the release. It would be rather simplified then.

**Review by eulertour — APPROVED** [review]

**Review by leotrs — APPROVED** [review]

**Pull request merged**